src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/config.jelly

```diff
@@ -36,9 +36,18 @@
                 <f:select/>
             </f:entry>
 
-            <f:entry title="${%Storage_Account_Name}" field="storageAccountName"
-                     help="/plugin/azure-vm-agents/help-storageAccountName.html">
-                <f:textbox/>
+            <f:entry title="${%Storage_Account_Name}">
+                <f:radioBlock name="storageAccountNameReferenceType" value="new" title="${%Storage_Account_Create_New}" checked="${instance == null || instance.isCreateNewStorageAccount('new')}" inline="true">
```

Comment:
Looks like there is a implicit type conversion from `String` to `bool`. Easy to have bugs since string`false` in JavaScript will implicitly be `true` in if expression. Not sure how jelly handle this case. But I suggest using something like `...isCreateNewStorageAccount == "true"` if we cannot update the return type of this method in backend.